### PR TITLE
add test that checks minting second asset calcs

### PR DESCRIPTION
**Pull request by goncer**

### CODE REVIEW

**Comment by goncer** [comment]
hey @PetoU  
does this test suits your expectation?
It basically:
* Enters an amount in the user's liquidity asset pool
* Validates that the autofilled amount is right
I'll add more cases as soon as we have locatos and/or the bug of displaying names for new assets is solved.

![screenshot_UI tests - Get Tokens from Faucet As a User I get the right values for Pool investment -  object Object]
(https://user-images.githubusercontent.com/15139384/128516029-71c2184e-79b7-42a7-a539-9089a9df3747.png)

**Comment by goncer** [comment]
@Andkoo  @omaksi , 
This FE test is about minting liquidity. When I enter a value into one asset, how is calculated the required second asset value? calculate_buy_price?
Wondering If there is an specific calcs that needs to be tested deeply.
Thanks!

File: test/ui/ui-accuracy.pool.investing.test.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        testUser1.addFromMnemonic(keyring, mnemonic);
+
+        sudo = new User(keyring, sudoUserName);
+        await sudo.mint(new BN(0),testUser1,new BN(visibleValueNumber));
+	});
+    beforeEach( async () => {
+        [firstCurrency, secondCurrency] = await Assets.setupUserWithCurrencies(testUser1, [new BN(visibleValueNumber),new BN(visibleValueNumber)], sudo);
+
+    })
+    ///TODO: Extend when locators available.Right now I can not select more liq. pools- I dont have Ids, and I dont have asset names :S
```
Comment (Andkoo) [inline test/ui/ui-accuracy.pool.investing.test.ts:59]: FYI: if "locators" means the `data-testid` element accessors I've added - they are available now.

**Review by Andkoo — APPROVED** [review]
LGTM

File: test/ui/ui-accuracy.pool.investing.test.ts (hunk 1 above)
Comment (goncer) [reply test/ui/ui-accuracy.pool.investing.test.ts:59]: yes, but AFIK they are not deployed in staging, and merging this PR would make all tests failing.
We can merge this one and later add more cases + fix locators.

**Pull request merged**